plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java

```diff
@@ -62,6 +64,9 @@ public MavenAppEngineStandardWizardPage() {
 
   @Override
   public void createControl(Composite parent) {
+    AnalyticsPingManager.getInstance().sendPing(
+        AnalyticsEvents.APP_ENGINE_NEW_PROJECT_WIZARD, null, null, parent.getShell());
```

Comment:
I suspect we should have separate events for the regular new project wizard and the maven new project wizard, or some sort of parameter that enables us to distinguish them